examples/src/test/java/com/amazonaws/examples/AwsKmsMultiRegionKeyGlobal2017IT.java

```diff
@@ -0,0 +1,20 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.amazonaws.examples;
+
+import static com.amazonaws.examples.TestUtils.US_EAST_1_MRK_KEY_ID;
+import static com.amazonaws.examples.TestUtils.US_WEST_2_MRK_KEY_ID;
+
+import java.security.GeneralSecurityException;
+import org.testng.annotations.Test;
+
+public class AwsKmsMultiRegionKeyGlobal2017IT {
+  private static final String TABLE_NAME = "tonyknap-manual-mrk-global-2017";
```

Comment:
more generic table name, since this goes to our examples? "I think TableName" or "SuperCoolTable" are worthy candidates?

Reply:
I do not think we ever going to ship this... but if we did, yes.